### DISCUSSION TITLE
Characters in Komga

### DISCUSSION THREAD

**Discussion by MKH-42** [gotson/komga#1551] [Ideas]
It would be nice to have characters like tags in komga.
It should also imported from comicinfo.xml

At the moment I use the tag flag to store characters in komga.
I found out, this is after a while and with a lot of characters not usefull because the idea behind tags to have quick access to special documents is not valid any more.

Why I want characters?
Special in older european comics from the 1960s, 1970s and 1980s there are a lot of collection comics.
In one comic there are a collecten from different comics, comic series and short stories.
e.g. in one comic are sigurd, vailant and tarzan. in the next comic tarzan, jan, spirou. the next comic includes sigurd and spirou and so on.
aditional this comics are not in one series. so it is hard to find out.

I also tried to create reading lists. but characters is the better way, goes faster and is in comicinfo.xml available.

But now the character list destroy my tag list so it would be nice to have 2 seperate lists. a character and a tag list.  

This requirement could be splitted in 2 parts:
a) to have charakters in komga 
-  fill this field manuelly over the ui and read the character tag from comicinfo.xml
- show the characters on the ui, comma separted in a nice format
- make it searchable so it is possible to search a series for e.g. sigurd.
b) optional: to create a filter list with this data set.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by MKH-42** [comment]
enrich the requirement.
now there are 2 parts:  to have characters in komga and a filter list (optional)

**Comment by CapitanoNemo78** [comment]
Useful to track in which comic you can already find those characters

**Comment by Irodzuita** [comment]
This would still be super helpful in 2024, my tags are extremely clogged with character/parody tags. With characters now being in the Anansi Project schema it would be nice for it to have its own section. For parody tags I think that using the Locations part of the schema would make sense (as a parody tag is usually just listing the world in which the story is taking place) and would really help to cleanup the tag list.

**Comment by OddMagnet** [comment]
I agree as well.

I switched from Kavita to Komga because I want to OAuth2 logins for my users, really missing the ability to easily created collections by filtering for characters though

**Comment by SteveXu9102** [comment]
This is also pretty helpful when there are doujinshis (fan comics) in the library.

**Comment by zndrr** [comment]
Also went looking for this feature. Hopefully it gets enough attention to be reconsidered.

**Comment by gotson** [comment]
Superseded by https://github.com/gotson/komga/discussions/2198
